src/main/java/com/backbase/communication/model/Email.java

```diff
@@ -0,0 +1,22 @@
+package com.backbase.communication.model;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Data
+public class Email {
```

Comment:
This is an intermediate model and it is suitable while we want to support multiple schemas, otherwise we can skip this model and just use generated one